www/docs/deploying.md

```diff
@@ -54,10 +54,27 @@ The `acl` and `use_backend` lines in the frontend configuration forward any traf
 Then, that backend rewrites those public-facing urls to the form expected by the reactive service (in this case, just by prepending `/v1`) and specifies the reactive service's address: `reactive_service:8080` here for a service defined as `reactive_service` using Docker Compose and using the default port.
 This is a simple "hello world" configuration for a single reactive service; in practice, more involved URL rewriting schemes and ACLs can be employed to orchestrate traffic between multiple public HTTPS endpoints and/or backend reactive services.
 
-### Control operations and reads/writes
+## Control operations and reads/writes
 
 The control port of your Skip reactive service should in general only be exposed to other backend services within your secure network, since it allows access to stream creation/deletion and data read/write operations.
 
 This also allows a Skip reactive service to be integrated into your existing web backend(s) -- regardless of their choice of programming language or web framework.
 For example, see [here](https://github.com/SkipLabs/skip/tree/main/examples/hackernews/web_service/app.py) for an example Python web app using Flask and backed by a reactive service which it communicates to directly over HTTP using the `requests` library, and [here](https://github.com/SkipLabs/skip/tree/main/skipruntime-ts/examples) (at `*-server.ts`) for a variety of examples in JavaScript using Express.
 For JavaScript backends, we also provide a programmatic interface [`SkipServiceBroker`](api/helpers/classes/SkipServiceBroker) which simplifies the integration and obviates the need to construct and manipulate HTTP requests directly.
+
+## Horizontal scaling
+
+For Skip services with sustained or bursty high-throughput requirements, Skip reactive services can scale horizontally to serve many clients without overloading a single machine or incurring heavy resource costs.
+
+The most common bottleneck for reactive systems is *memory*, due to the heavy use of caching and materialization of intermediate results.
+In a Skip service, much of this memory pressure comes from instantiating large numbers of resources to stream customized/personalized data to many clients.
+To handle that scenario, you can run your Skip service in a *leader-follower* topology, running:
+
+    * one *leader* instance to maintain the shared computation graph and/or pull in data from external dependencies like databases or polled APIs, and
+    * one or more *followers* which synchronize that shared data from the leader, among which resource instances are distributed in a round-robin fashion.
```

Comment:
AFAIU "round-robin" is just an implementation detail of the particular example, not inherent in any way, right?